src/components/Station.tsx

```diff
@@ -6,6 +6,7 @@ import { getStationById } from '../helpers/ApiCallHelper';
 interface Station {
     id: number;
     name: string;
+    displayName?: string;
```

Comment:
leaving property "displayName" unused for now because "name" suffices - it's in the interface just to replicate all the data available and may be used in the future